package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.0.3",
   "description": "a CSS component library for TeamSnap",
   "main": "dist/js/index.js",
+  "sideEffects": false,
```

Comment:
What does this address?

Reply:
I was doing some treeshaking testing with Webpack 4 (forgot to remove from this branch).  Since we are currently still building with babel, this can be removed.  Good catch.   

In general in case you were curious - https://medium.com/webpack/road-to-webpack-4-week-20-21-1641d03ce06e